srml/aura/src/lib.rs

```diff
@@ -75,8 +128,10 @@ impl InherentDataProvider {
 	}
 }
 
+/// Methods to specify the provision of inherent data
 #[cfg(feature = "std")]
 impl ProvideInherentData for InherentDataProvider {
+	/// Called when this inherent data provider is registered at the given `InherentDataProviders`
```

Comment:
same for methods, except if it actually explains the implementation, but should probably in inner doc `//` because user doesn't want to care